test/unitTests/fakes.ts

```diff
@@ -221,6 +214,48 @@ export function getFakeVsCode(): vscode.vscode {
                 throw new Error('Not Implemented');
             },
         },
+        l10n: {
+            t: (
+                options:
+                    | string
+                    | {
+                          message: string;
+                          args?: Array<string | number | boolean> | Record<string, any>;
+                          comment: string | string[];
+                      },
+                ...args: (string | number | boolean)[] | Record<string, any>[]
+            ) => {
```

Comment:
Yes - apparently this is ho9w you have to implement an interface that defines overloads - define one function that handles every possible variation on the input.  Quite insane really.